### PR TITLE
index react-popper docs

### PR DESCRIPTION
**Pull request by FezVrasta**
# Pull request motivation(s)

I added a new section about the react-popper documentation and I need it to be indexed.

### CODE REVIEW

File: configs/popper.json [hunk 1]
```diff
@@ -4,16 +4,23 @@
...
     },
-    "https://popper.js.org/docs/"
+    {
+      "url": "https://popper.js.org/react-popper/v2/",
+      "tags": [
+        "react-popper-v2"
+      ]
+    },
+    "https://popper.js.org/docs/",
+    "https://popper.js.org/react-popper/
```
Comment (s-pace) [inline configs/popper.json:23]:
```suggestion
    "https://popper.js.org/react-popper/"
```

**Comment by FezVrasta** [comment]
@s-pace for some reason the search doesn't return any result now, may you help me understand what's going on please?

**Comment by s-pace** [comment]
Tha correctly tag was not set up

**Comment by FezVrasta** [comment]
Oh right I did completely forget about what the tags were for...

**Pull request merged**